src/get_programming_with_js_next/module_6_exercise.es6

```diff
@@ -0,0 +1,15 @@
+/*
+  Write a function that will take an email address and mask all of the characters up to the @.
+
+  For example the email address dog@example.com would be masked to ***@example.com
+*/
+
+function maskEmail(address) {
+  let domainPosition = address.indexOf('@');
+
+  let domain = address.substr(domainPosition);
+
+  return domain.padStart(address.length, '*');
```

Comment:
`padStart` is an es7 feature, not yet implemented into browsers. It's cool to play with, though :)

Another possible approach:
```javascript
function maskEmail(address) {
  const [username, domain] = address.split('@');
  return `${username.split('').map(_ => '*').join('')}@${domain}`;
}
```